### PR TITLE
Added README to base-template and tweaked chunked list advice

### PR DESCRIPTION
**Pull request by camerons**
## Purpose / why

* README added based on feedback that it wasn't clear where to start with templates.
* chunked list advice is review of aidantwork 

## Verification

* Read new text, confirm it improves our story.
* This is not addressing an issue as I'm not using issues in the incubator during this development phase.

---

## Checklist

> __Pull-request reviewer__ should ensure the following

* [ ] Are issues linked correctly?
* [ ] Is this PR labeled correctly?
* [ ] If template updates: do they align with [developers.google.com/style/](https://developers.google.com/style/)?
* [ ] Did the PR receive at least one :+1: and no :-1: from core-maintainers?
* [ ] On merging, did you complete the merge using [keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue)?
* [ ] On merging, did you add any applicable notes to a [draft release](https://github.com/thegooddocsproject/templates/releases) and link to this PR?

### CODE REVIEW

File: base-template/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# What's in the base-template directory
+
+This base-template directory contains templates and guides to be followed by a `doctype` template author. They can also be used by a project's docset owner select and customize the project's templates.
```
Comment (aidanatwork) [inline base-template/README.md]: "docset owner select" should be changed to "docset owner to select"

File: base-template/base-doctype-guide-template.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ _{TBD: We should link to a page we are yet to write helping people pick personas
...
+* For content with numerous items or steps, consider "chunking" content into sub-sections of 5-10 steps. It makes the information easier to read and remember, and gives the reader a sense of accomplishment after each chunk is completed. Chunking is recommended by major companies, such as within [Microsoft's writing style guide](https://docs.microsoft.com/en-us/style-guide/procedures-instructions/writing-step-by-step-instructions#complex-procedures), and from the Nielsen Norman Group's [research on chunking and usability](https://www.nngroup.com/articles/short-term-memory-and-web-usability/).
```
Comment (aidanatwork) [inline base-template/base-doctype-guide-template.md]: "Chunking is recommended by major companies, such as within Microsoft's writing style guide" would perhaps read better as "Chunking is recommended by major companies, such as Microsoft in its writing style guide"

**Review by aidanatwork — CHANGES_REQUESTED** [review]
@camerons, I made a couple minor suggestions for changes, for improved readability.

File: base-template/README.md (hunk 1 above)
Comment (camerons) [reply base-template/README.md]: Thanks, fixed.

File: base-template/base-doctype-guide-template.md (hunk 2 above)
Comment (camerons) [reply base-template/base-doctype-guide-template.md]: Good suggestion. Fixed.

**Pull request merged**